accord-core/src/main/java/accord/utils/RandomSource.java

```diff
@@ -174,6 +180,79 @@ default DoubleStream doubles(double minInclusive, double maxExclusive)
 
     double nextGaussian();
 
+    default int pickInt(int first, int second, int... rest)
```

Comment:
I'm not keen on introducing a method to a non-test class that is inefficiently implemented like this.

Reply:
> I'm not keen on introducing a method to a non-test class that is inefficiently implemented like this

can you elaborate, I am not sure at what your comment is getting to; are you asking for something like the below?

```
        int offset = nextInt(0, rest.length + 2);
        switch (offset)
        {
            case 0: return first;
            case 1: return second;
            default:
                return rest[offset - 2];
        }
```

> Also, this doesn't need the Int suffix. This feedback applies to the Long and T variants.

It does, else java can struggle in a few cases; these used to be called `pick` until javac complained about not knowing what I wanted.  In this patch I think it doesn't matter, but this was pulled from CASSANDRA-18675 in which this `javac` issue was faced.